README.md

```diff
@@ -100,7 +100,7 @@ $ npm install -g @devcycle/cli
 $ dvc COMMAND
 running command...
 $ dvc (--version)
-@devcycle/cli/5.11.1 linux-x64 node-v18.16.0
```

Comment:
are we supposed to be committing these changes? my env always generates them but i just delete them